cmd/searcher/search/store.go

```diff
@@ -120,8 +124,13 @@ func (s *Store) prepareZip(ctx context.Context, repo gitserver.Repo, commit api.
 		return "", errors.Errorf("commit must be resolved (repo=%q, commit=%q)", repo.Name, commit)
 	}
 
+	largeFilePatterns := conf.Get().SearchLargeFiles
+	lfpBytes, err := json.Marshal(largeFilePatterns)
+	if err != nil {
+		return "", errors.Errorf("error marshalling large file patterns: %v", err)
+	}
 	// key is a sha256 hash since we want to use it for the disk name
-	h := sha256.Sum256([]byte(string(repo.Name) + " " + string(commit)))
+	h := sha256.Sum256([]byte(string(repo.Name) + " " + string(commit) + " " + string(lfpBytes)))
```

Comment:
You can avoid the marshalling and have a simpler implementation:

```suggestion
	h := sha256.Sum256([]byte(fmt.Sprintf("%q %q %q", repo.Name, commit, conf.Get().SearchLargeFiles)))
```

Reply:
Oooooh fancy. Didn't know about `%q`